airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -243,20 +243,17 @@ def __update_running_task(self, task):
         task_key = self.active_workers.arn_to_key[task.task_arn]
 
         # Mark finished tasks as either a success/failure.
-        if task_state == State.FAILED:
-            self.fail(task_key)
+        if task_state == State.FAILED or task_state == State.REMOVED:
             self.__log_container_failures(task_arn=task.task_arn)
-        elif task_state == State.SUCCESS:
-            self.success(task_key)
-        elif task_state == State.REMOVED:
             self.__handle_failed_task(task.task_arn, task.stopped_reason)
-        if task_state in (State.FAILED, State.SUCCESS):
+        elif task_state == State.SUCCESS:
```

Comment:
Why don't we want to send the below debug message for Failed tasks anymore?

Reply:
Perhaps I'm being overly cautious of backwards compatibility, this is still experimental after all.